lib/modules/platform/gitea/index.md

```diff
@@ -4,6 +4,17 @@ Renovate uses modern Git upload filters to suppress large blob downloads.
 For Gitea you need to manually enable upload filters.
 Read the official [Gitea docs](https://docs.gitea.io/en-us/clone-filters/) for more information.
 
+## Authentication
+
+First, [create a Personal Access Token](https://docs.gitea.io/en-us/api-usage/#authentication-via-the-api) for the bot account.
```

Comment:
```suggestion
First, [create a Personal Access Token](https://docs.gitea.io/en-us/api-usage/#authentication) for the bot account.
```

The old link "fragment" does not exist anymore, I think we should link to this fragment instead.